src/seedu/addressbook/parser/Parser.java

```diff
@@ -75,33 +82,37 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
```

Comment:
the "case" should be align with "switch". Note that sometimes the coding style in Intellij is different from what we expect.